src/src-assets/player-assets/player.coffee

```diff
@@ -427,9 +420,12 @@ angular.module('Adventure', ['ngAria', 'ngSanitize'])
 	# Do stuff when the user submits something in the SA answer box
 	$scope.handleShortAnswerInput = ->
 
-		response = $scope.response
+		response = originalResponse = $scope.response
 		$scope.response = ""
 
+		matches = []
+		selectedMatch = null
+
 		# Outer loop - loop through every answer set (index 0 is always [All Other Answers] )
 		for i in [0...$scope.q_data.answers.length]
 
```

Comment:
Just noticed we need to reset `response` to the original response at the start of each loop iteration. There's an edge case here where if a preceding answer doesn't require case sensitivity, then `response` will be lowercase forever, breaking the case sensitivity option for following answers.

Reply:
Great catch, fixed!